### PR TITLE
Mod Organizer support in editor

### PR DESCRIPTION
**Pull request by domportera**
A simple tool for debugging mods and mod compatibility, and also profiling daggerfall engine functions that the mods rely on. Efforts were made to make this simple to use and largely non-destructive

I'm new to ModOrganizer (and this repo of course. and daggerfall) so I don't know if this will work with non-portable Mod Organizer data folders. I only have familiarity with a portable data folder. Though in theory it should.

### CODE REVIEW

File: Assets/Game/Addons/ModSupport/Editor/ModOrganizerSupport/ModOrganizerMenuItems.cs [hunk 1]
```diff
@@ -16,11 +17,13 @@ internal class ModOrganizerMenuItems
...
-        static void Select()
+        static async void Select()
         {
             Debug.Log($"<color=orange>Select your Mod Organizer data directory.</color> " +
                       $"This path should contain your `profiles` and `mods` subfolders.");
 
+            await Task.Delay(500);
+
             string dataPath = EditorUtility.OpenFolderPanel("Select Mod Organizer data path",
                 String.Empty, String.Empty);
```
Comment (KABoissonneault) [inline Assets/Game/Addons/ModSupport/Editor/ModOrganizerSupport/ModOrganizerMenuItems.cs:28]: Maybe the UX move would be not to mix console output with modal windows. If a window needs more context before being shown, then you should make the message a modal popup

Comment (domportera) [reply Assets/Game/Addons/ModSupport/Editor/ModOrganizerSupport/ModOrganizerMenuItems.cs:28]: I'd normally agree but that's just an extra thing to click away once you learn what to do after the first time you do it. I agree this isn't terribly pretty but it seems the most comfortable to me. I'm open to ideas though

**Comment by Interkarma** [comment]
Hi, please be aware that Daggerfall Unity is about to enter Release Candidate. To maintain stability and mod compatibility leading up to 1.0, I am no longer accepting new Pull Requests other than direct bug fixes. All changes that do not fix an accepted issue will not be merged.

This ensures I have time to complete my personal commitments to core project, rather than only servicing requests from mod authors. Once Daggerfall Unity reaches 1.0, I will be stepping away from this project to work on something new. All Pull Requests not accepted at that time will be closed. For more details, please see below topic on forums.

https://forums.dfworkshop.net/viewtopic.php?t=6018

Thank you for your contribution and understanding.

-Interkarma

**Comment by domportera** [comment]
> Hi, please be aware that Daggerfall Unity is about to enter Release Candidate. To maintain stability and mod compatibility leading up to 1.0, I am no longer accepting new Pull Requests other than direct bug fixes...

That's perfectly valid, thank you for all the work you've put into it thus far

**Pull request closed without merging**